docs/install/install-server/install-server.inc

```diff
@@ -59,36 +59,102 @@ Install the software
         The first package (inmanta-oss) contains all the code and the commands. The server and the agent packages install config
         files and systemd unit files. The dashboard is installed with the server package.
 
-    .. tab:: Other Linux and Mac
+
+    .. tab:: Debian, Ubuntu and derivatives. 
 
         First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
         Next install inmanta with pip install in the newly created virtual env.
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
-            # Install python3 >= 3.6 and git
+            # Install GCC, Python pip and Wheel
+            sudo apt-get update
+            sudo apt-get install build-essential
+            sudo apt-get install python3-pip
+
+            pip3 install wheel
+            # OR
+            pip install wheel
+
+
+            # Install python3 >= 3.6, venv and git
+            sudo apt-get install python3-venv
             sudo python3 -m venv /opt/inmanta
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
+            
+            # Install PostgreSQL
+            sudo apt-get install postgresql postgresql-client
+
+
+        Download the configuration file named ``inmanta.cfg`` (this name is arbitrary) in your virtual env:
+
+        .. code-block:: sh
+
+            sudo apt-get install wget
+            sudo wget -O /opt/inmanta/inmanta.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/inmanta.cfg"
+
+
+        If you want to use the dashboard you need to install it as well:
+
+        Get the pre-built package from `our github page <https://github.com/inmanta/inmanta-dashboard/packages/>`_ click on the latest release and then on the right hand side, under ``Assets``, you will see the compressed package, download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under ``[dashboard]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
+
+        .. code-block:: bash
+
+            path=/opt/inmanta/package/dist
+
+
+        Then the dashboard can be started using below command (please note, below command has to be run after completing the ``Configure server`` :ref:`Configure server` ) part:
```

Comment:
When I compile the documentation, I get the warning: `install/install-server/install-server.inc:107: WARNING: undefined label: configure server`.

Reply:
Fixed it.